Trainer_v4/Trainer_v4.csproj

```diff
@@ -43,36 +43,30 @@
     <Compile Include="Utilities.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="Libraries\Assembly-CSharp-firstpass.dll" />
-    <Content Include="Libraries\Assembly-CSharp.dll" />
-    <Content Include="Libraries\UnityEngine.CoreModule.dll" />
-    <Content Include="Libraries\UnityEngine.dll" />
-    <Content Include="Libraries\UnityEngine.UI.dll" />
-  </ItemGroup>
-  <ItemGroup>
+    <Folder Include="Libraries\" />
     <Folder Include="Properties\" />
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>Libraries\Assembly-CSharp.dll</HintPath>
+      <HintPath>..\..\..\Software Inc_Data\Managed\Assembly-CSharp.dll</HintPath>
```

Comment:
I don't agree with this approach since you are forcing developer to have project files close to the game install. Even my approach isn't the best because you don't have always the newest libraries. I'm leaving this as an open question which should be discussed.

Reply:
I re-instated the libraries for now. I honestly forgot to branch off while making those changes. I was of the mind that the easiest and laziest way of making modifications/coding was to put the git repo into ${gamedir}/DLLmods

With that done the game would compile it for you, and that puts the mod geographically very close to the latest version of the libraries.